travvy-server/src/config/config.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = {
+  port: 3306,
```

Comment:
if this doesnt work, not sure if localhost:3306 would change anything, but 3306 should be the right port

Reply:
Yeah I just checked, 3306 is the port I used to connect last time. This should be good